src/cfp.md

```diff
@@ -2,7 +2,11 @@
 layout: default
 title: Call for Proposals
 ---
-
+<div class="row">
+  <div class="box box_red">
+    <h2 class="centred">Our Call for Proposals will open later in the year.</h2>
+  </div>
+</div>
```

Comment:
Adding the `centred` class will only work if there are styles for that CSS class defined in our project, and I don't think there are. Try something like:

```suggestion
<style>
#cfp_alert {
  text-align: center;
}

#cfp_alert h2 {
  margin-bottom: 0;
}
</style>

<div class="row">
  <div class="box box_red" id="cfp_alert">
    <h2 class="centred">Our Call for Proposals will open later in the year.</h2>
  </div>
</div>
```

Reply:
Would you prefer this or for me to add stuff to the .scss files?

Reply:
SCSS is nicer, but the site is small enough that it’s not a big deal either way. I suggested it inline because that makes it a bit easier to experiment with.